AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Requests/SendRequestStatusToGetASmokeAlarmHandlerShould.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using AllReady.Models;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Requests
+{
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+
+    using AllReady.Areas.Admin.Features.Requests;
+    using AllReady.Hangfire.Jobs;
+
+    using global::Hangfire;
+    using global::Hangfire.Common;
+    using global::Hangfire.States;
+
+    using Moq;
+
+    public class SendRequestStatusToGetASmokeAlarmHandlerShould : InMemoryContextTest
+    {
+        private readonly Guid requestId;
+
+        public SendRequestStatusToGetASmokeAlarmHandlerShould()
+        {
+            this.requestId = Guid.NewGuid();
+            this.Context.Requests.Add(new Request
+            {
+                RequestId = this.requestId,
+                Source = RequestSource.Api
+            });
+            this.Context.SaveChanges();
+        }
+
+        [Fact]
+        public async Task NotQueueUpAnythingIfTheRequestStatusIsNotASupportedStatus()
+        {
+            var notification = new RequestStatusChangedNotification
+            {
+                RequestId = this.requestId,
+                NewStatus = RequestStatus.PendingConfirmation
+            };
+
+            var backgroundJobClient = new Mock<IBackgroundJobClient>();
+            var sut = new SendRequestStatusToGetASmokeAlarmHandler(this.Context, backgroundJobClient.Object);
+
+            await sut.Handle(notification);
+
+            backgroundJobClient.Verify(x => x.Create(It.IsAny<Job>(), It.IsAny<EnqueuedState>()), Times.Never);
+        }
+
+        [Fact]
+        public async Task NotQueueAnythingIfTheRequestCannotBeFound()
+        {
+            var notification = new RequestStatusChangedNotification
+            {
+                RequestId = Guid.NewGuid(),
+                NewStatus = RequestStatus.Completed
+            };
+
+            var backgroundJobClient = new Mock<IBackgroundJobClient>();
+            var sut = new SendRequestStatusToGetASmokeAlarmHandler(this.Context, backgroundJobClient.Object);
+
+            await sut.Handle(notification);
+
+            backgroundJobClient.Verify(x => x.Create(It.IsAny<Job>(), It.IsAny<EnqueuedState>()), Times.Never);
+        }
+
+        [Theory]
+        [InlineData(RequestStatus.Completed)]
```

Comment:
nice use of `[Theory]` and `[InlineData]` here. For what the class is doing, using the xUnit tools makes perfect sense here.